crates/sui-core/src/traffic_controller/policies.rs

```diff
@@ -25,6 +28,12 @@ enum ClientType {
 #[derive(Hash, Eq, PartialEq, Debug)]
 struct SketchKey(IpAddr, ClientType);
 
+struct HighestRates {
+    direct: BinaryHeap<Reverse<(IpAddr, u64)>>,
+    proxied: BinaryHeap<Reverse<(IpAddr, u64)>>,
```

Comment:
Shouldn't these be `Reverse<(u64, IpAddr)>`? You want to sort by rate first, right?

Reply:
Ah yep, thanks